Quaternion.mpl

```diff
@@ -66,11 +71,54 @@ quaternion: [{
   entries: new;
 }];
 
+quaternion: [
+  m:;
+  m matrix?
+  m getColCount 3 = and
+  m getRowCount 3 = and
+] [
+  trans quaternionFromRotationMatrix
+] pfunc;
+
 quaternionCast: [
   q: virtual Schema: Ref;;
   q.entries [@Schema cast] (each) quaternion
 ];
 
+quaternionFromRotationMatrix: [
```

Comment:
Why do we need a separate method here?

Reply:
https://github.com/Matway/mpl-sl/pull/30#discussion_r932126679
So it is clearer that this function takes rotation matrix and quaternion takes transposed one. You can also use this function separately